src/main/java/step3/CarRacingApplication.java

```diff
@@ -0,0 +1,19 @@
+package step3;
+
+public class CarRacingApplication {
+    public static void main(String[] args) {
+        InputView inputView = new InputView();
+        int carCount = inputView.carCount();
+        int tryCount = inputView.tryCount();
+
+        RacingGame racingGame = new RacingGame();
+        racingGame.readyCar(carCount);
+
+        System.out.println("실행결과");
+        for (int i = 0; i < tryCount; i++) {
+            ResultView.gameResultView(racingGame.playGame());
```

Comment:
```suggestion
        for (int i = 0; i < tryCount; i++) {
            racingGame.playGame();
            ResultView.gameResultView(racingGame.getCars());
```

상태를 변경하는 메서드와 결과를 반환하는 메서드를 분리시켜서 사용할 수 있을 것 같네요
(메서드 명은 예시로 작성했으니 참고해주세요!)